### DISCUSSION TITLE
打包发布到微软商店？

### DISCUSSION THREAD

**Discussion by nynauy** [HMBSbige/NatTypeTester#245] [Ideas]
目前微软商店里应该没有同类产品，请问作者是否考虑将其打包发布到商店？
如果作者没有这个打算的话，我是否可以在注明原始版权声明和许可信息的前提下，以NatTypeTester作为应用名称发布到微软商店？

**Comment by HMBSbige** [comment]
之后可能考虑自己上传（等完善了
